packages/database/src/prisma-api.ts

```diff
@@ -380,13 +380,14 @@ export function getSdk(
       // Note: if the user already has a bulk purchase/coupon, then if they are
       // only adding 1 seat to the team, then it is still a "bulk purchase" and
       // we need to add it to their existing Bulk Coupon.
-      const isBulkPurchase = quantity > 1 || !!existingBulkCoupon
+      const isBulkPurchase = quantity > 1 || Boolean(existingBulkCoupon)
 
       let bulkCouponId = null
       let coupon = null
 
       if (isBulkPurchase) {
-        bulkCouponId = existingBulkCoupon ? existingBulkCoupon.id : v4()
+        bulkCouponId =
+          existingBulkCoupon !== null ? existingBulkCoupon.id : v4()
```

Comment:
No behavioral change here, I just wanted to clean this up by making these values coerce to booleans.